### ISSUE TITLE
[PAPER] Unsupervised Representation Learning with Deep Convolutional Generative Adversarial Networks

### ISSUE THREAD

**Issue by dailyhenie** [HephaestusProject/pytorch-DCGAN#1]
## 📝 Introduction

* 논문 분야: Computer Vision 
* 논문에서 정의한 문제: GAN 학습의 불안정성, CNN 을 활용한 unsupervised learning 연구의 부족
* 논문에서 제안한 방법론: CNN 을 활용하여 GAN 학습 안정화
   - all convolutional net 
   - eliminating fully connected layer on top of convolutional features
   - batch normalization
   - ReLU in the generator, LeakyReLU in the discriminator

## Why?
생성모델의 가장 기본이 되는 모델로, Generation Model 을 시작하기 가장 좋은 논문이라고 생각했습니다. 

## Issue card & Schedule

* 데이터셋 준비
* 테스트 코드 작성 
* 모델 구현 
* Pytorch-lightning
* W&B 
* 서빙 

## Reference
[Unsupervised Representation Learning with Deep Convolutional Generative Adversarial Networks](https://arxiv.org/pdf/1511.06434.pdf)

**Comment by ssaru** [comment]
오! GAN! GAN을 하시는 이유가 있으실까요?
시작하게된 계기가 궁금하네요! ㅎㅎ

**Comment by dailyhenie** [comment]
@ssaru 
전에 얘기했듯이 생성모델 쪽 (스타일 트랜스퍼 계열 모델)을 이용해서 회사에 도움 될 만한 모델 하나를 만들어보고 싶었어요 그래서 완전 기초가 되는 논문 하나를 선택했구요!!

**Comment by ssaru** [comment]
@hyerim1048 답변감사합니다. 화이팅이예요! ㅎㅎ
